### ISSUE TITLE
Is this library working properly?

### ISSUE THREAD

**Issue by rvision** [imwh0im/dayjs-range-extend#7]
How is it possible that those 2 ranges overlap?

```
const d1 = dayjsRange('2022-01-01', '2022-01-05');
const d2 = dayjsRange('2022-01-05', '2022-01-10');

console.log(d1.isOverlap(d2)); // => true
```

**Comment by imwh0im** [comment]
@rvision 
```ts
const d1 = dayjsRange('2022-01-01', '2022-01-05'); // => '2022-01-01 00:00:00' ~ '2022-01-05 00:00:00'
const d2 = dayjsRange('2022-01-05', '2022-01-10'); // => '2022-01-05 00:00:00' ~ '2022-01-10 00:00:00'
```
overlap d1's `endDate` and d2's `startDate`.

**Comment by rvision** [comment]
@imwh0im 

But they shouldn't - they do not overlap, first range ends when second one starts.

**Comment by rvision** [comment]
```
import Moment from "moment";
import { extendMoment } from "moment-range";

const moment = extendMoment(Moment);

const range1 = moment.range(moment("2022-01-01"), moment("2022-01-05"));
const range2 = moment.range(moment("2022-01-05"), moment("2022-01-10"));

const overlaps = range1.overlaps(range2);
console.log(overlaps); // => false
```

**Comment by imwh0im** [comment]
@rvision 
Thank you for your feedback.
I think it needs to be modified. i can do it myself or you can give me a pull-request. what do you want a be?

**Comment by rvision** [comment]
@imwh0im 

`return this.startDate < other.endDate && this.endDate > other.startDate;`

I need this for range calculations, and overlapping is the one I am after, but I am not sure if I should use the library, this is the basic functionality expected and it is not working properly.

**Comment by imwh0im** [comment]
@rvision 
check this please #8

**Comment by imwh0im** [comment]
any things else needs test case?

**Comment by imwh0im** [comment]
@rvision 
Released v1.0.4
check [npm](https://www.npmjs.com/package/dayjs-range-extend)
